src/pages/Clients.tsx

```diff
@@ -0,0 +1,55 @@
+import AppContainer from "../components/layout/AppContainer";
+import { Flex, Button, Input } from "@mantine/core";
+import { INPUTS_CLIENTS_PAGE } from "../constants/clients/clients";
+import CustomInput from "../components/shared/CustomInput";
+import { IconSearch, IconX } from "@tabler/icons-react";
+import CustomTable from "../components/shared/CustomTable";
+
+import {
+  CustomMainTitle,
+  ClientsAddingPanel,
+  ClientsInputsGroup,
+} from "../styles/ClientsPage";
+
+const Clients = () => {
+  return (
+    <AppContainer>
+      <CustomMainTitle order={1}>Клиенты</CustomMainTitle>
```

Comment:
```suggestion
      <Title mb="30px" order={1}>Клиенты</Title>
```
why not use [style props](https://mantine.dev/styles/style-props/#supported-props)
add translate for this and throughout app